crates/cargo-contract/src/main.rs

```diff
@@ -17,6 +17,7 @@
 #![deny(unused_crate_dependencies)]
 
 mod cmd;
+mod config;
```

Comment:
This could be moved inside the `cmd` module since it appears only to be used there

Reply:
Done